.github/workflows/website.yaml

```
@@ -51,6 +51,8 @@ jobs:
             docusaurus.config.ts
       - name: Build
         working-directory: website
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Could you use `GH_TOKEN` like `gh` does?

```suggestion
          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```